scripts/taxonomy/classification.py

```diff
@@ -68,6 +56,9 @@ def main(
     material_mapping = getHierarchyMapping("material")
     specimen_mapping = getHierarchyMapping("specimen")
 
+    MetadataModelLoader.initialize_models()
```

Comment:
this is a driver class that tests out the classifiers by applying it to database records.

Reply:
(And this piece feels like it properly belongs in scripts)